glide.lock

```diff
@@ -0,0 +1,60 @@
+hash: 24dd85605b7c94a4ecc2c7fd16496787124c613e5fe272ec102575af738b01c0
```

Comment:
i dont htink this file should be version controlled. Can you add a globbing exclusion of .lock file in .gitignore pls